### PR TITLE
add IDEA configuration matching Spotify checkstyle

### PR DESCRIPTION
**Pull request by pettermahlen**
I don't understand these config files very well (and don't particularly feel like I want to spend the time ;) ), but this is a version that at least solves the checkstyle warnings I was getting from one particular project.

Comments/ideas about fixing? Shall we add this and improve gradually?

### CODE REVIEW

**Comment by pettermahlen** [comment]
@mattnworb @jo-ri comments, maybe?

**Comment by mattnworb** [comment]
I don't understand the config itself that well either but this sounds sensible to me if it works for you 👍

File: src/main/idea/spotify-checkstyle-idea.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    <option name="DOWHILE_BRACE_FORCE" value="3" />
+    <option name="WHILE_BRACE_FORCE" value="3" />
+    <option name="FOR_BRACE_FORCE" value="3" />
+    <indentOptions>
+      <option name="INDENT_SIZE" value="2" />
+      <option name="CONTINUATION_INDENT_SIZE" value="4" />
+      <option name="TAB_SIZE" value="8" />
+    </indentOptions>
+  </codeStyleSettings>
+  <codeStyleSettings language="JavaScript">
```
Comment (mattnworb) [inline src/main/idea/spotify-checkstyle-idea.xml]: is it intentional to add JavaScript settings too?

Comment (pettermahlen) [reply src/main/idea/spotify-checkstyle-idea.xml]: No. :) Removed those, and some other settings that seem to be wrong.

**Pull request merged**